drivers/ws281x/include/ws281x_params.h

```diff
@@ -64,6 +64,47 @@ static const ws281x_params_t ws281x_params[] =
     WS281X_PARAMS
 };
 
+/** @brief Timer used for WS281x (by the timer_gpio_ll implementation)
+ *
+ * A single timer is configured for any number of WS281x strands, so this does
+ * not need to be part of params.
+ *
+ * It is required that the timer has at least 2 channels. (Future versions may
+ * require a 3rd channel).
+ *
+ * It is required that the timer's MAX_VALUE is 2^n-1, which is a trivial but
+ * not explicitly stated case.
+ *
+ * This timer is configured at WS281x initialization time, and kept stopped
+ * outside of transmissions.
+ *
+ * The default value of 2 is chosen because the only platform on which the
+ * module is usable is nRF5x, where TIMER_DEV(1) is in use by the radio module.
+ * It is strongly advised to explicitly set this timer to a known free timer,
+ * as the default may change without notice.
+ * */
+#if !defined(WS281X_TIMER_DEV) || defined(DOXYGEN)
+#define WS281X_TIMER_DEV TIMER_DEV(2)
```

Comment:
This needs to be overwritten e.g. for boards using nRF52810xxaa, as they only seem to have 2 timers. We typically do that by adding a `define` in the board's `board.h`.